lib/web/join_tokens.go

```diff
@@ -53,6 +53,7 @@ import (
 
 const (
 	stableCloudChannelRepo = "stable/cloud"
+	HeaderTokenName        = "X-TOKEN-NAME"
```

Comment:
Should this be something like `X-Teleport-TokenName`?

Reply:
yes it should